src/ray/common/id.h

```diff
@@ -78,78 +96,209 @@ class UniqueID : public BaseID<UniqueID> {
 
 class JobID : public BaseID<JobID> {
  public:
-  static constexpr int64_t length = 4;
+  static constexpr int64_t kLength = 4;
 
-  // TODO(qwang): Use `uint32_t` to store the data.
   static JobID FromInt(uint32_t value);
 
-  static size_t Size() { return length; }
+  static size_t Size() { return kLength; }
 
   static JobID FromRandom() = delete;
 
   JobID() : BaseID() {}
 
  private:
-  uint8_t id_[length];
+  uint8_t id_[kLength];
+};
+
+class ActorID : public BaseID<ActorID> {
+ private:
+  static constexpr size_t kUniqueBytesLength = 4;
+
+ public:
+  /// Length of `ActorID` in bytes.
+  static constexpr size_t kLength = kUniqueBytesLength + JobID::kLength;
+
+  /// Size of `ActorID` in bytes.
+  ///
+  /// \return Void
```

Comment:
```suggestion
  /// \return Size of `ActorID` in bytes.
```
For simple getters like this, I think it's also okay to just use the one-liner and drop the `\return` statement.